pkgs/tools/security/doas/default.nix

```diff
@@ -34,8 +31,14 @@ stdenv.mkDerivation rec {
     ./0001-add-NixOS-specific-dirs-to-safe-PATH.patch
   ];
 
+  # Standard environment supports "dontDisableStatic" knob, but has no
+  # equivalent for "--disable-shared", so I have to patch "configure"
+  # script instead.
   postPatch = ''
     sed -i '/\(chown\|chmod\)/d' GNUmakefile
+    sed -i '/unknown option/d' configure
```

Comment:
Is there any chance you could create a patch for the configure file and just add `--disable-shared` as a no-op flag, instead of removing the "error on unknown option" catchall branch?